package.json

```diff
@@ -160,7 +159,9 @@
     "superagent": "^3.8.2",
     "ts-node": "^3.3.0",
     "tslint": "^5.8.0",
+    "tslint-config-prettier": "^1.15.0",
     "typescript": "^2.7.2",
-    "webpack": "^3.5.5"
+    "webpack": "^3.5.5",
+    "winston": "2.4.0"
```

Comment:
I'm probably missing something - what's the reason for adding more logging dependencies here?

Reply:
Yeah, that looks to be a mistake - good pickup.